.circleci/config.yml

```diff
@@ -506,7 +506,9 @@ jobs:
 
       - run:
           name: Build Android RNTester App
-          command: ./gradlew RNTester:android:app:assembleRelease
+          command: |
+            ./gradlew RNTester:android:app:assembleDebug
+            ./gradlew RNTester:android:app:assembleRelease
```

Comment:
Looks like we merged the other changes in this PR [here](https://github.com/facebook/react-native/pull/27417) but not this config update, is this something we need as well? cc @mdvacca